bundles/com.amazonaws.eclipse.cloudformation/src/com/amazonaws/eclipse/explorer/cloudformation/wizard/CreateStackWizardFirstPage.java

```diff
@@ -445,8 +445,13 @@ public Object convert(Object fromObject) {
                 if ( "None".equals(value) ) {
                     return 0;
                 } else {
-                    String minutes = value.substring(0, value.indexOf(' '));
-                    return Integer.parseInt(minutes);
+                    Integer a = value.indexOf(' '); 
+                    if (a != -1)
+                    {
+                        String minutes = value.substring(0, );

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;,&#34;, Expression expected after this token